### PR TITLE
Implement Helmholtz Corporate Design

### LINKED ISSUE #3: Add an imprint
Webpages in Germany are legally required to provide an imprint.

### LINKED ISSUE #4: Redesign /software
Redesign the software list so it will be shown in Helmholtz Corporate Design.

### LINKED ISSUE #5: Redesign /projects
Redesign the projects list so it will be shown in Helmholtz Corporate Design.

### LINKED ISSUE #6: Redesign /organisations
Redesign the organisations list so it will be shown in Helmholtz Corporate Design.

### LINKED ISSUE #7: Redesign /software/add
Redesign the add-software page so it will be shown in Helmholtz Corporate Design.

### LINKED ISSUE #8: Redesign /software/[slug]/edit
Redesign the software edit page so it will be shown in Helmholtz Corporate Design.

### LINKED ISSUE #9: Redesign /projects/add
Redesign the "Add projects" page so it will be shown in Helmholtz Corporate Design.

### LINKED ISSUE #10: Redesign /projects/[slug]/edit
Redesign the "Edit projects" page so it will be shown in Helmholtz Corporate Design.

### LINKED ISSUE #11: Data protection declaration
Add a link to the data protection declaration by GFZ.

### LINKED ISSUE #14: Spotlights on landing page
Implement a box on the frontpage that

* shows the latest spotlight (=featured software) in a large box
* some previous spotlights in three or four smaller boxes below

Here is a small sketch showing the idea

image

It would be great to have a background image for the boxes. However, there is currently no way to add an image to a piece of software. This may have to be added upstream.

### PR DESCRIPTION
**Pull request by cmeessen**
Implements the following features:

* Imprint, closes #3 
* Data protection declaration, closes #11
* Helmholtz Corporate Design / Landing page, addresses #2
* Spotlights on landing page, closes #14 
* Helmholtz Corporate Design / all other pages, closes #4, closes #5, closes #6, closes #7, closes #8, closes #9, closes #10

How to test:

* pull branch and recurse submodules
  ```
  git checkout 14-spotlights_landingpage --recurse-submodules
  ```
* copy `.env.example` to `.env` and fill in the required information
  * remember to add yourself to `RSD_ADMIN_EMAIL_LIST`
  * set `RSD_AUTH_PROVIDERS=HELMHOLTZAAI`
  * if you need the AAI secret, ask @ruester or @cmeessen 
* build
  ```
  make dev
  ```
* if you want to migrate some data, go to [this repo](https://git.gfz-potsdam.de/id2/hifis/rsd/spotlight-migration), clone it to a different location, and follow the instructions to migrate the latest software spotlights to your local RSD
* access the RSD via http://localhost:3000

To do's:

* [x] Finalise imprint
* [x] Add hard coded software spotlights on the front page
* [x] Provide text content for the front page

PR Checklist:

*   [ ] Increase version numbers in `docker-compose.yml`
*   [x] Link to a GitHub issue
*   [ ] Update documentation
*   [ ] Tests

### CODE REVIEW

**Comment by Normo** [comment]
This still needs to be translated:
german_translation

File: frontend/pages/index.tsx [hunk 1]
```diff
@@ -48,228 +47,265 @@ type HomeProps = {
...
-                Register your <br/> Organisation
+        {/* Head and claim */}
+        <div className="bg-secondary bg-landing-page">
+          <div className="flex flex-row flex-wrap container mx-auto px-6 md:px-10 pt-16 pb-12 max-w-screen-xl text-white">
+            <div className="min-w-min flex flex-col">
+              <LogoHelmholtz width="220" />
+              <div className="pt-2 pb-12">Spitzenforschung für<br />große Herausforderungen.</div>
+              <a onClick={handleClickOpen}>
+                <div className="w-[250px] bg-[#05e5ba] hover:bg-primary text-secondary hover:text-white text-center font-medium text-2xl py-4 px-6 rounded-sm">
+                  Add your software
```
Comment (Normo) [inline frontend/pages/index.tsx:208]: Noticed during testing: Prior authentication this button finally redirects to the Software overview. After being authenticated the button does nothing.

**Comment by cmeessen** [comment]
@Normo thanks, I missed that. It's fixed now (force push).

File: frontend/pages/index.tsx [hunk 2]
```diff
@@ -48,228 +47,265 @@ type HomeProps = {
...
+              <div className="grid grid-cols-1 md:grid-cols-2 gap-8 place-items-center pt-8 text-black">
+                <a href="https://esciencecenter.nl" className="hover:cursor-pointer w-full grid place-items-center" target="_blank" rel="noreferrer">
+                  <LogoEscience width="100%"/>
+                </a>
+                <a href="https://hifis.net" className="hover:cursor-pointer w-full grid place-items-center" target="_blank" rel="noreferrer">
+                  <LogoHifis width="60%" />
+                </a>
               </div>
-            </div>
+            </div> */}
```
Comment (Normo) [inline frontend/pages/index.tsx:265]: Still needed?

File: frontend/pages/index.tsx [hunk 3]
```diff
@@ -48,228 +47,265 @@ type HomeProps = {
...
+
+        {/* Roadmap */}
+        {/* <div className="bg-white">
+          <div className="conainer mx-auto py-6 px-4 max-w-screen-xl text-secondary">
+            <h1 className="pb-4">Roadmap</h1>
+            <p className="prose">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Proin libero nunc consequat interdum. Pretium vulputate sapien nec sagittis aliquam malesuada bibendum arcu. Id volutpat lacus laoreet non curabitur gravida arcu. Amet mauris commodo quis imperdiet massa tincidunt nunc. Varius sit amet mattis vulputate. Suscipit adipiscing bibendum est ultricies integer. Hendrerit gravida rutrum quisque non tellus. Eget felis eget nunc lobortis mattis aliquam. Integer enim neque volutpat ac tincidunt vitae. Condimentum id venenatis a condimentum vitae sapien pellentesque habitant.</p>
+          </div>
+        </div> */}
+
+        <AppFooter/>
```
Comment (Normo) [inline frontend/pages/index.tsx:308]: Stil needed?

Comment (Normo) [inline frontend/pages/index.tsx:308]: #2 mentions *Participating Centres* and *Partner Organisations* as requirements. They are still missing.

**Review by Normo — CHANGES_REQUESTED** [review]
Great work @cmeessen 
I've reviewed it and added some comments. Please have a look.

File: frontend/pages/index.tsx (hunk 2 above)
Comment (cmeessen) [reply frontend/pages/index.tsx:265]: Yes, let's keep it for now, until we have the final front page.

File: frontend/pages/index.tsx (hunk 3 above)
Comment (cmeessen) [reply frontend/pages/index.tsx:308]: See above

Comment (cmeessen) [reply frontend/pages/index.tsx:308]: Fair point, I am not sure how to implement the organisations and centres on the frontpage. They are shown on the "Organisations" page. Let's not close #2 with this PR.

**Pull request merged**